dataset.proto

```diff
@@ -27,6 +28,13 @@ message DataSet {
     Status            error = 4;
   }
 
+  message Language {
```

Comment:
Why are there reserved fields in a new message? I'm wondering if we should also more formally define what "code" is. Looking at this as a model: https://godoc.org/golang.org/x/text/language#Tag (which in turn uses https://tools.ietf.org/html/bcp47). Not sure we need to change anything to support, but we might want some documentation about it.